tutor/src/components/icon.cjsx

```diff
@@ -30,13 +30,13 @@ module.exports = React.createClass
       'clickable': @props.onClick or (@props.tooltip and @props.tooltipProps.trigger is 'click')
     })
 
-    icon = <i {...@props} className={classNames} />
-
-    if @props.tooltip
-      classNames =
-      tooltip = <BS.Tooltip id={@state.uniqueId}
-        className={classnames({'on-navbar': @props.onNavbar})}
-        >{@props.tooltip}</BS.Tooltip>
-      <BS.OverlayTrigger {...@props.tooltipProps} overlay={tooltip}>{icon}</BS.OverlayTrigger>
-    else
-      icon
+    if not @props.tooltip
+      return <i {...@props} className={classNames} />
+
+    icon = <button {...@props} className={classNames} />
+    tooltip = <BS.Tooltip id={@state.uniqueId}
+      className={classnames({'on-navbar': @props.onNavbar})}
+      >{@props.tooltip}</BS.Tooltip>
+
+    <BS.OverlayTrigger {...@props.tooltipProps} overlay={tooltip}>{icon}</BS.OverlayTrigger>
+
```

Comment:
Good idea to use button!  I think the non-existent "OX Styleguide" says to use `unless` instead of `if not` (line 33) though 😁